infra/cmake/packages/CMSIS-NN-6.0.0/CMSIS-NNConfig.cmake

```diff
@@ -0,0 +1,14 @@
+function(_CMSIS_NN_import)
+    nnas_include(ExternalSourceTools)
+    nnas_include(OptionTools)
+
+    envoption(EXTERNAL_DOWNLOAD_SERVER "https://github.com")
+    envoption(CMSIS_NN_6_0_0_URL ${EXTERNAL_DOWNLOAD_SERVER}/ARM-software/CMSIS-NN/archive/refs/tags/v6.0.0.tar.gz)
```

Comment:
@BalyshevArtem this PR does not work with CMSIS_NN 4.1 ? I am not sure which CMSIS_NN version in TizenRT.

Reply:
Hmmm, if it is critical, I can rewrite it with 4.1 cmsis-nn, but current implementation required 6.0. TFLITE-micro uses the 6.0 version. Rewrite using 4.0?

Reply:
It does not matter if it is compatible to TizenRT(but I do not know which version in TizenRT in use. It 'was' compatible to 4.1)

Reply:
FYI, internal TizenRT use the same file at https://github.com/Samsung/TizenRT/blob/master/external/include/cmsis_nn/arm_nnfunctions.h#L1675 for cmsis_nn. Seem like it is based on 6.0 instead of 4.1

Reply:
>  Rewrite using 4.0?

Is it possible ? I do not know about SVDF. But CMSIS_NN 4.1 does not use `kernel_sum` at all in SVDF kernel.

Reply:
> Is it possible ?
Yes :)
Okay, I will rewrite to 4.1 and close prs with cmsis-nn version updating

Reply:
Done, ready for review